MainApp/templates/MainApp/profile.html

```diff
@@ -140,13 +140,49 @@ <h3 style="color: rgb(99, 99, 99);">@{{account.following.username}}</h3>
     {% endfor %}
 
   {% elif clickedtab == 4 %} <!-- Settings -->
+
     <div class="post">
       <form method="POST">
         {% csrf_token %}
         {{ form.as_p }}
         <button name="submit_user_edits" type="submit" value=1 >Save User </button>
       </form>
     </div>
+
+  {% elif clickedtab == 5 %} <!-- Likes -->
+
+    {% for post,isLiked in liked_tweet_obj_dict.items %}
+    {% if forloop.counter0 < 25 %}
+    {% load nbsp %}
+      <div class="post">
+      <!--<div class="fakeimg" style="width: 10%; height: 100%; float: left;">Place Holder</div>-->
+      <div class="defaultimg" style="height:10%;width:10%; float:left;"> <img src = "https://simplyilm.com/wp-content/uploads/2017/08/temporary-profile-placeholder-1.jpg" alt="DefaultPic" width="70" height="70"></div>
+      <div style="float: left; width: 90%; padding-left: 10px;">
+        <span style="font-size: 20px; font-weight: bold;">{{post.tweet_creator.first_name}} {{post.tweet_creator.last_name}}</span>
+        <span>
+          <a class="user_link" href="/profile/{{post.tweet_creator.username}}/">@{{post.tweet_creator.username}} </a>
+        </span>
+        <span style="font-size: 20px; color: rgb(99, 99, 99);"> - {{post.pub_date}}</span>
+        <div style="word-wrap: break-word; padding-top: 5px; line-height: 130%;">{{ post.visible_info | linebreaksbr | nbsp }}</div>
+        <div class="fakeimg" style="height:60px;">Image</div>
+      </div>
+      <!--If the post user equals the authenticated user, do not show like button-->
+      {% ifnotequal user.username post.tweet_creator.username %}
+      <form name="like_tweet" method="POST" action="{% url 'MainApp:likestab' user %}">
+        {% csrf_token %}
+        <!--Show either like or unlike depending on whether or not user has liked the tweet-->
+        {% if not isLiked %}
+          <button name="like_button" value="{{user.username}},{{post.pk}}" class="like">Like</button>
+          {% else %}
+            <button name="unlike_button" value="{{user.username}},{{post.pk}}" class="like">Unlike</button>
+          {% endif %}
+        </form>
+        {% endifnotequal %}
+        <div style="clear: both;"></div>
+      </div>
+    {% endif %}
+    {% endfor %}
+
```

Comment:
Added a like tab page that shows all of the tweets that a user has liked...they can also unlike those tweets right from this page